src/main/java/com/example/school/SchoolApplication.java

```diff
@@ -212,6 +214,21 @@ private static List<Student> findBestTruantsInSchool(List<Activity> activities)
                 .collect(Collectors.toList());
     }
 
+    // method that gets a list of students from the group who skipped the most lessons
+    private static List<Student> findBestTruantsInGroup(List<Activity> activities, Group group) {
+        var amountSkippedLessonsForEachStudentFromSchool = countSkippedLessonsForEachStudent(activities);
+        var amountSkippedLessonsForEachStudentFromGroup = amountSkippedLessonsForEachStudentFromSchool.entrySet().stream()
+                .filter(e -> e.getKey().getGroup().equals(group))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
```

Comment:
Let's do next:
1) filter `activities` by group id;
2) send filtered `activities` to `countSkippedLessonsForEachStudent(...)`

Reply:
I can't filter activities by group id, because there is not group in activity.